src/media/js/routes.js

```diff
@@ -72,5 +73,6 @@ define('routes',
         'search': search,
         'settings': '/api/v2/account/settings/mine/',
         'site-config': '/api/v2/services/config/site/?cache=1&serializer=commonplace&vary=0',
+        'website': '/api/v2/websites/website/{0}/?cache=1&vary=0',
```

Comment:
Aaand I just noticed I had not registered the view in zamboni, so that's not working at the moment. https://bugzilla.mozilla.org/show_bug.cgi?id=1165373